### PR TITLE
Fixes #10422

### LINKED ISSUE #10422: Surgical switchtool funny goings on
#### Description of issue

When you attempt to put a deployed surgical switchtool onto a table, it puts the deployed tool (for instance, if you were in cautery mode, it'd put a cautery on the table), the switchtool would remain in your hands, and undeploy itself.

The lost tool that had then been put onto the table was lost from the list of possible tools usable by the switchtool.
#### Difference between expected and actual behavior

Expected - For it to remain deployed and put on the table

Actual - I lost a tool from my compact tool-carrier
#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by ghost**
Fixes #10422

### CODE REVIEW

**Comment by ComicIronic** [comment]
Does this mean you can drop the switchtool on the table now?

Also fix the atmos PDA part of that same issue.

**Comment by ghost** [comment]
@ComicIronic Can't place a deployed switchtool on a table. Not entirely sure how I'd fix that.
Device Analyser PDA fixed. You can place a PDA with the device analyser on the table.

**Comment by clusterfack** [comment]
I'm fairly sure there's a way if you toy with the logic of the preattack a bit

**Comment by ghost** [comment]
@clusterfack beats me.

**Comment by ComicIronic** [comment]
You only fixed the device analyzer and not the atmos analyser that also exists in the PDA.

**Comment by ghost** [comment]
Whoops.

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -2211,6 +2212,7 @@ obj/item/device/pda/AltClick()
 /obj/item/device/pda/afterattack(atom/A as mob|obj|turf|area, mob/user as mob)
 	if(scanmode == 5)
```
Comment (clusterfack) [inline code/game/objects/items/devices/PDA/PDA.dm]: wooow magic number

**Comment by ComicIronic** [comment]
We'll have to figure out a fix for not being able to drop the switchtool onto a table directly, but that can wait.

**Pull request merged**